src/main/java/seedu/address/logic/commands/addcommands/AddInternshipCommand.java

```diff
@@ -69,7 +73,14 @@ public CommandResult execute(Model model) throws CommandException {
         }
 
         companyToAddTo.addInternship(internship);
-        return new RegularCommandResult(String.format(MESSAGE_SUCCESS, Messages.formatInternship(internship)));
+
+        companyToAddTo.updateFilteredInternshipList(PREDICATE_SHOW_ALL_INTERNSHIPS);
```

Comment:
Sorry I'm not too sure what this does, does `PREDICATE_SHOW_ALL_INTERNSHIPS` show all companies with internships?

Reply:
It shows all the internships for a company.

Reply:
Is there a situation in which all internships will not be shown?